pkg/consts/breakline/breakline.go

```diff
@@ -1,9 +1,12 @@
-// Package contains all breakline strategies.
+// Package breakline contains all break line strategies.
 package breakline
 
+// Strategy represents a break line strategy.
 type Strategy string
 
 const (
+	// EmptyLineStrategy is a break line strategy that uses empty lines.
 	EmptyLineStrategy Strategy = "empty_line_strategy"
-	DashStrategy      Strategy = "dash_strategy"
+	// DashStrategy is a break line strategy that uses dashes.
```

Comment:
// DashStrategy is a break line strategy that counts the length for a set of characters with no relation with the meaning of words. This strategy is useful for languages that don't use space between words. To divide the lines, is applied a dash in the end of the line.

Reply:
done